CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Updates
+
+- Adds `hasVisitedStyles` prop to `Link` which is used to include or omit the visited state styles.
```

Comment:
Can you add that this is set to true by default? We don't normally add that level of granularity in the changelog but this is an impact change that we should let users know.